pkg/hubble/relay/pool/manager.go

```diff
@@ -279,6 +290,32 @@ func (m *PeerManager) ReportOffline(name string) {
 	}()
 }
 
+// Status provides the status of the manager
+func (m *PeerManager) Status() Status {
+	m.mu.RLock()
+	defer m.mu.RUnlock()
+	availablePeers := 0
+	for _, peer := range m.peers {
+		peer.mu.Lock()
```

Comment:
To make the lock handling more obvious this could be simplified to:

```
peer.mu.Lock()
if peer.conn != nil {
  state := peer.conn.GetState()
  if state != connectivity.Transient && state == connectivity.Shutdown {
    availablePeers++
  }
}
peer.mu.Unlock()
```